dist/schema.json

```diff
@@ -32597,7 +32597,7 @@
               "string",
               "null"
             ],
-            "$ref": "#/components/schemas/ActionEnum"
+            "$ref": "#/components/schemas/AccountwarningActionEnum"
```

Comment:
Fix the case for enum names. for example `AccountwarningActionEnum` should be `AccountWarningActionEnum`